pkg/client/client.go

```diff
@@ -297,3 +297,16 @@ func DefaultOpenShiftUserAgent() string {
 	version = seg[0]
 	return fmt.Sprintf("%s/%s (%s/%s) openshift/%s", path.Base(os.Args[0]), version, runtime.GOOS, runtime.GOARCH, commit)
 }
+
+type HybridClient struct {
```

Comment:
> @deads2k joint client?

Yes.  `HybridClient` should not exist.  You should specify what you want in your API.